ios/Sources/SettingFeature/SettingScreen.swift

```diff
@@ -46,41 +46,40 @@ public struct SettingScreen: View {
     }
 
     public var body: some View {
-        NavigationView {
```

Comment:
I deleted NavigationView because it is stacked doubly when transitioning to SettingScreen and its display gets collupted.

Reply:
OK, thank you.